applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -109,6 +114,43 @@ void rsp_send(const u8_t *str, size_t len)
 	}
 }
 
+static int set_uart_baudrate(u32_t baudrate)
+{
+	int err = -EINVAL;
+	struct uart_config cfg;
+
+	LOG_DBG("Set uart baudrate to:%d", baudrate);
+	if (baudrate == 0) {
```

Comment:
Do we need to check for `baudrate == 0` explicitly here? 0 is as bad as any other incorrect value. Plus the input validation was already done in `handle_at_slmuart` function.

Reply:
Yes. I've removed the redundant check.